R/plot_variance.R

```diff
@@ -0,0 +1,66 @@
+#' Plot as a heatmap the scaled between imputation variance for every cell
+#'
+#' This function plots the cell-level between imputation variance. The function
+#' scales the variances column-wise and plots the data image as a heatmap.
+#' Darker red cells indicate more variance, lighter cells indicate less variance.
+#' white cells represent observed cells or unobserved cells with zero between
+#' imputation variance.
+#'
+#' @param object A `mice` generated multiply imputed data set of class `mids`.
+#'
+#' @return An object of class `ggplot`.
+#' @importFrom dplyr mutate
+#' @importFrom dplyr across
+#' @importFrom dplyr everything
+#' @importFrom ggplot2 aes
+#' @importFrom mice mdc
+#' @importFrom dplyr select
+#' @importFrom dplyr group_by
+#' @importFrom dplyr summarise
+#' @importFrom dplyr ungroup
+#' @importFrom tidyr pivot_longer
+#' @importFrom stats var
+#' @examples
+#' library(mice)
+#' library(dplyr)
+#' library(tidyr)
+#' library(ggplot2)
+#' imp <- mice(nhanes)
+#' plot_variance(imp)
+#' @export
+plot_variance <- function(object){
```

Comment:
Other `ggmice` functions use `data` as the first argument.

Reply:
will change